backend/worker.go

```diff
@@ -90,23 +90,20 @@ func (w *worker) Start(ctx context.Context) {
 	// TODO: Check for already started worker
 	ctx, cancel := context.WithCancel(ctx)
 	w.cancel = cancel
-
+	b := &backoff.ExponentialBackOff{
```

Comment:
Consider passing the context in the backoff object, so we can avoid the `select` below that checks if the context has been canceled. See: https://pkg.go.dev/github.com/cenkalti/backoff/v4#WithContext

Reply:
used it